proxy/.gitignore

```diff
@@ -1,2 +1,4 @@
 # direnv (https://direnv.net/)
 .envrc
+
+private
```

Comment:
On my machine, I put an copy of the old `config.toml` here and will use `--config private/config.toml` from now on. You could do the same. But I think we do not want actual providers in our default `config.toml`.

Reply:
> On my machine, I put a copy of the old `config.toml` (with the providers I use) here and will use `--config private/config.toml` from now on. 

Doesn't git allow a global `.gitignore` in the home directory for such use cases? Otherwise you would need to add personal stuff that is not related to the project (IDE config, etc.) to all local `.gitignore` files.

> You could do the same. But I think we do not want actual providers in our default `config.toml`.

Why not? It's convenient. IMO convenience is more important than cleanness here.

Reply:
> Doesn't git allow a global .gitignore in the home directory for such use cases? Otherwise you would need to add personal stuff that is not related to the project (IDE config, etc.) to all local .gitignore files.

Yes, it does. I kinda prefer local `.gitignores` when the folder at hand is something that could as well make sense on it's own. It also keeps effects local. Maybe it's a matter of taste. Do you prefer to change it?

> Why not? It's convenient. IMO convenience is more important than cleanness here.

I agree that it's convinient for us :-) But it could also raise some questions. Why is it GMX and not provider X? Is this project affiliated with Mailbox.org? I would prefer to not have existing providers it in the default config.

Reply:
> Do you prefer to change it?

It doesn't bother me (maybe later when I create a rust module named `private` and will be confused...). I just thought that you are spending too much time managing and explaining me your `.gitignore` file.

> I agree that it's convinient for us :-) But it could raise some questions. Why is it GMX and not provider X? Is this project affiliated with Mailbox.org? Thus, I would prefer to not have existing providers in the default config.

I'd prefer to have them all :p

Reply:
> I'd prefer to have them all :p

Any idea how to resolve this dilemma? 🙈 

I would be okay with an additional comment in the config. But "These are here because the devs use them for testing" is kinda... not right.

Reply:
I don't mind enough to resist any longer, so feel free to ignore my comment.

And I'm sure we are not the only devs that are using these free providers for testing :p